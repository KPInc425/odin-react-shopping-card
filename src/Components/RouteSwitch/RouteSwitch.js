import { BrowserRouter, Routes, Route, Link } from 'react-router-dom';
import App from '../App/App';
import Nav from '../Nav/Nav';
import ShoppingCart from '../ShoppinCart/ShoppingCart';


const RouteSwitch = () => {
    return (
        <BrowserRouter >
            <Nav /> 
            <Routes>
                <Route path='/' element={ <App />} />
                <Route path='/shoppingcart' element={ <ShoppingCart /> } />
            </Routes>
        </BrowserRouter>
    )
}


export default RouteSwitch