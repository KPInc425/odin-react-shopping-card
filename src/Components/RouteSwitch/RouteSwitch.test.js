import { render, screen } from '@testing-library/react';
import RouteSwitch from './RouteSwitch';

describe("RouteSwitch Component", () => {
    it("renders route browser", () => {
        const { container } = render(<RouteSwitch />);
        expect(container).toMatchSnapshot();
    });
})