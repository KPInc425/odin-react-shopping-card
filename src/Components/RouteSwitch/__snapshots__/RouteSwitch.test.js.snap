// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouteSwitch Component renders route browser 1`] = `
<div>
  <nav>
    <ul>
      <li>
        Menu Item 1
      </li>
    </ul>
  </nav>
  <div
    class="App"
  >
    <h1>
      Home Page
    </h1>
  </div>
</div>
`;
